templates/addon/depends/common/{{ game.name }}/{{ game.name }}.txt.j2

```diff
@@ -1 +1 @@
-{{ game.name }} https://github.com/libretro/{{ repo | default(game.name) }} {{ repo_branch | default('master') }}
+{{ game.name }} https://github.com/libretro/{{ repo | default(game.name) }} {{ repo_hexsha | default('repo_branch') | default('master') }}
```

Comment:
This only works as long as a tag or branch points to exactly this commit.

Reply:
As already said, it only works as long as a tag or branch points to exactly to the referenced commit hash.

Currently master branch points to that hash, but once libretro pushes a commit to their repo the hash won't be cloned with depth of 1 anymore.
see https://github.com/xbmc/xbmc/blob/0963f41/cmake/scripts/common/HandleDepends.cmake#L190
You can try yourself what happens by writing the parent hash of master branch into the file.

Reply:
Sorry I overlooked that it just works now because master points the change.
What would you suggest? Just leave it master? Or can we figure out if the commit hash doesn't point to a tag/branch and remove the GIT_SHALLOW in that case?

Reply:
Just because libretro can't correctly version their cores, we won't do the same thing. So keeping it to master isn't acceptable.

Best thing would be if they start versioning correctly, but as I fear that won't happen #36 is an option.

Reply:
@Rechi I doubt that they'll start tagging versions any time soon. I'll update the code to download `SHA.zip`.
There's still one core where we can't use this due to git submodules, but at least it helps us to get the majority of addons published in the repo.

Could you answer the question below? Start pkg version from 0, 1 or 100?

Reply:
Start with 0, 1, 100 or something else, doesn't really matter, only makes me wondering what happened with 0 - 99.